src/sentry/monitors/tasks/detect_broken_monitor_envs.py

```diff
@@ -86,7 +119,53 @@ def detect_broken_monitor_envs():
                 monitor_incident=open_incident, defaults={"detection_timestamp": current_time}
             )
             if not detection.user_notified_timestamp:
-                # Record that we need to notify users about this broken detection via email
-                pass
+                environment_name = open_incident.monitor_environment.get_environment().name
+                project = Project.objects.get_from_cache(id=open_incident.monitor.project_id)
+                for user in project.member_set:
+                    if not user.user_email:
+                        continue
+
+                    user_monitor_entry = user_broken_envs[user.user_email][
+                        open_incident.monitor.slug
+                    ]
```

Comment:
Is there a chance the same user could have multiple broken monitors with the same slug across different projects?

Monitors are unique per-project now, instead of per-org, so is it safe to index on slug here?

Reply:
ahh you're right, maybe its best to just index on monitor guid and store slug as a dictionary value instead